src/m365/spfx/commands/spfx-doctor.ts

```diff
@@ -701,44 +703,41 @@ class SpfxDoctorCommand extends AnonymousCommand {
     return (<any>SharePointVersion)[sp.toUpperCase()];
   }
 
-  private getSharePointFrameworkVersion(logger: Logger): Promise<string> {
-    return new Promise<string>((resolve: (version: string) => void, reject: (error: string) => void): void => {
+  private async getSharePointFrameworkVersion(logger: Logger): Promise<string> {
+    if (this.projectRootPath !== null) {
+      const spfxVersion = this.getProjectVersion();
+      if (spfxVersion) {
+        return spfxVersion;
+      }
+    }
```

Comment:
Let's add a debug message here when we find the SPFx version this way

Reply:
Great suggestion. I added something like this:
![image](https://user-images.githubusercontent.com/9193870/224180418-f5be54d1-2071-4a12-b4f5-c80ca5c35bb5.png)
Would that suffice?

Reply:
Perfect!